### PR TITLE
Flux group keys demo

### PR DESCRIPTION
**Pull request by sanderson**
This PR introduces in new group key demo that lets doc users visually get sense of how Flux group keys work. This is part of the new getting started with Flux docs in the flux-restructure line of work.

group-key-demo

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/flux/v0.x/get-started.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Flux is a **functional data scripting** language designed to unify the processes
+of querying, processing, analyzing, and acting on data into a single syntax.
+
+## Data model
+### Stream of tables
+### Table
+### Column
+### Row
+### Group key
+
```
Comment (kelseiv) [inline content/flux/v0.x/get-started.md:20]: Maybe Table grouping example would work best under the Table header? 

May be helpful to introduce/highlight example data and purpose along the lines of: 

This example includes the following data:
- `example` measurement
- `loc` tag with two values
-  sensorID` tag with two value
- `templ` and `hum` fields with multiples values. 

By default, this example demonstrates how data is modeled if no group key is applied. To filter data by group key, select one or more of the check boxes (keys?) to see resulting table records/rows._

Any reason not to limit sample two values per field? ....shorter to scan and consume.

File: content/flux/v0.x/get-started.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{{< flux/group-key-demo >}}
+
+## Data types
+### Basic types
+### Composite types
```
Comment (kelseiv) [inline content/flux/v0.x/get-started.md:25]: same as above

**Review by kelseiv — APPROVED** [review]

File: content/flux/v0.x/get-started.md (hunk 1 above)
Comment (kelseiv) [reply content/flux/v0.x/get-started.md:20]: Any reason not to limit sample two values per field? ....shorter to scan and consume.

File: content/flux/v0.x/get-started.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Flux is a **functional data scripting** language designed to unify the processes
+of querying, processing, analyzing, and acting on data into a single syntax.
+
+## Data model
+### Stream of tables
+### Table
+### Column
+### Row
+### Group key
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/flux/v0.x/get-started.md:19]: Yeah, these are just placeholders.

File: content/flux/v0.x/get-started.md (hunk 1 above)
Comment (sanderson) [reply content/flux/v0.x/get-started.md:20]: > Any reason not to limit sample data to two values per field? ....shorter to scan and consume.

That's the reason 😄

File: content/flux/v0.x/get-started.md (hunk 2 above)
Comment (sanderson) [reply content/flux/v0.x/get-started.md:25]: Just place holders. These may end up being broken into multiple docs.

**Comment by jstirnaman** [comment]
Nice visual! General question: we're not using any javascript processing/transpilation pipeline, right?

**Review by jstirnaman — APPROVED** [review]
Nice visual! General question: we're not using any javascript processing/transpilation pipeline, right?

**Comment by sanderson** [comment]
@jstirnaman We're using Hugo's js/asset pipeline which essentially combines and minifies the js files. It's fairly rudimentary and was introduced in an earlier version of Hugo. I know they've made some updates to js processing, but I haven't taken the time to dig into the new functionality.

**Pull request merged**